source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/lib/index.ts

```diff
@@ -118,9 +166,10 @@ export class LambdaToElasticSearchAndKibana extends Construct {
       userpool: this.userPool,
       identitypool: this.identityPool,
       cognitoAuthorizedRoleARN: cognitoAuthorizedRole.roleArn,
-      serviceRoleARN: lambdaFunctionRoleARN}, props.esDomainProps);
```

Comment:
wow, this was cruddy formatting...

like playing hide and seek with the final parameter.